src/.vuepress/components/kistanExclusive.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+const props = defineProps<{
+    lmixer?: boolean
+}>();
+
+let header = "Kistan exclusive!";
+let description = "This content only applies to kistan";
```

Comment:
Uppercase the name `Kistan` in all places.

```suggestion
let description = "This content only applies to Kistan";
```